pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "mercury-engine-data-structures>=0.21.0",
+    "mercury-engine-data-structures>=0.23.3.dev44",
```

Comment:
uhh

Reply:
this is draft for a reason. I uploaded the dev as pre-release so I could run CI :P